assignments/arrays.js

```diff
@@ -64,33 +64,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+for(let i = 0; i < inventory.length; i++){
+    if(inventory[i].id === 33){
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}.`);
+        break;
+    }
+}
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
-
+for(let i = 0; i < inventory.length; i++){
+    carModels.push(inventory[i].car_model)
+}
+carModels.sort(function(a,b){
```

Comment:
Very clever use of custom `sort` function. The default behaviour of sort if no function is provided, is to sort alphabetically but it is case sensitive. Your solution ignores case but it also preserves the original casing on the array. Since you were a step ahead refer to the following code which also achieves the same result. Research about these functions in MDN.

```js
carModels.sort(function (a, b) {
  return a.localCompare(b, 'en', { sensitivity: 'base' });
}
```